Modules/Core/Common/include/itkHexahedronCell.hxx

```diff
@@ -356,7 +356,7 @@ HexahedronCell<TCellInterface>::EvaluatePosition(CoordRepType *            x,
     for (unsigned int i = 0; i < Self::NumberOfPoints; ++i)
     {
       pt = points->GetElement(m_PointIds[i]);
-      for (unsigned int j = 0; j < Self::CellDimension; ++j)
+      for (unsigned int j = 0; j < PointType::Dimension; ++j)
```

Comment:
Not all loops that index the `pt` variable have had `Self::CellDimension` changed to `PointType::Dimension`.  Also not changed are the definitions of variables (e.g., `fcol`, `rcol`, `scol`, and `tcol`) that are also used in those loops.  Should they be similarly changed?  (Or more globally, but with much more speculation on my part, should the constexpr definition of `CellDimension` be changed to be `PointType::Dimension`?)

Reply:
@Leengit I don't know this code at all.  These are the only problems that were exposed during compilation.  

Do you know this code? Could you fix this better?

Reply:
Requested changes are outside the scope of this patch set.  They should be made into an issue and addressed by someone who knows the HexahedronCell code better.